### DISCUSSION TITLE
AttributeError: type object 'Config' has no attribute 'configure_pipeline'

### DISCUSSION THREAD

**Discussion by fbenav** [Avaiga/taipy#862] [Q&A]
Im triying to replicate the tutorials found here: https://docs.taipy.io/en/release-1.1/getting_started/step_05/ReadMe/

when i trying to replicate step 5, i got the error on the title

i tried on python 3.8 and 3.10

**Comment by trgiangdo** [comment]
The documentation page you were following is for Taipy 1.1.

For the up-to-date tutorials, please take a look at this page on the latest release of Taipy

https://docs.taipy.io/en/latest/knowledge_base/
